### ISSUE TITLE
I have the Batch 4 Lower Frame Tube Connector Files...but can't upload them here.

### ISSUE THREAD

**Issue by jamespkeane** [openppg/Mechanical#4]

**Comment by davek79** [comment]
If you want to post the files on the forum I can put them in the github.

**Comment by jamespkeane** [comment]
Dave,

I posted them with my post last night.  Did they not upload?

James Keane
jamespkeane@gmail.com
631.704.4596

________________________________

**Comment by davek79** [comment]
Posted to github.

**Comment by jamespkeane** [comment]
Great,

One issue I'm seeing is when I print them they seem to be printing
13.2-13.3 mm wide.  This is presenting an issue for me.  I'm trying to
troubleshoot it.  Wrote Paul to see if perhaps I got an outdate file.

Best regards,
James Keane
+1-631-704-4596
<https://www.linkedin.com/in/jpkeane>   [image: jamespkeane@gmail.com]
<jamespkeane@gmail.com>

**Comment by davek79** [comment]
Yeah, that’s a question for Paul, I’m unable to help with that unfortunately.

**Comment by jamespkeane** [comment]
I’m printing them with a scaled x axis now.  If that works I’ll send you new files to replace what I sent.

James Keane
jamespkeane@gmail.com
631.704.4596

________________________________

**Comment by jamespkeane** [comment]
Dave,

Attached is the final, fully dimensioned and fitting file!

Best regards,
James Keane
+1-631-704-4596
<https://www.linkedin.com/in/jpkeane>   [image: jamespkeane@gmail.com]
<jamespkeane@gmail.com>
